web/docs/guides/auth/auth-bitbucket.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: auth-bitbucket
+title: "Setting up Bitbucket Auth"
+description: Add Bitbucket OAuth to your Supabase project
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Setting up Bitbucket logins for your application consists of 3 parts:
+
+* Create and configure a Bitbucket OAuth Consumer on [Bitbucket](https://bitbucket.org)
+* Add your Bitbucket OAuth Consumer keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Steps
+
+### Log into your [Bitbucket](https://bitbucket.org) Account
+
+Once logged into *"Bitbucket"*:
+
+- Click on your profile icon at the bottom left
+- Click on "All Workspaces"
```

Comment:
If we are describing an element or section, we should make it a code item. eg: `All Workspaces` instead of "All Workspaces"